apps/src/lib/levelbuilder/NewCourseFields.jsx

```diff
@@ -0,0 +1,145 @@
+import React, {useState} from 'react';
+import HelpTip from '@cdo/apps/lib/ui/HelpTip';
+import PropTypes from 'prop-types';
+
+export default function NewCourseFields(props) {
+  const [selectedFamilyName, setSelectedFamilyName] = useState('');
+  const [newFamilyName, setNewFamilyName] = useState('');
+  const [versionedCourse, setVersionedCourse] = useState('');
+
+  return (
+    <div>
+      <label>
+        What family is this course a part of?
+        <select
+          value={selectedFamilyName}
+          style={styles.dropdown}
+          className="familyNameSelector"
+          onChange={e => {
+            setSelectedFamilyName(e.target.value);
+            props.setFamilyName(e.target.value);
+          }}
+          disabled={newFamilyName !== ''}
+        >
+          <option value={''}>(None)</option>
+          {props.families.map(familyOption => (
+            <option key={familyOption} value={familyOption}>
+              {familyOption}
```

Comment:
would it make sense to exclude family names where the existing unit in that family is "unversioned"? I'm not sure exactly what the right logic would be, but it seems like we could remove a lot of options from this list which the user would / should never pick.

Reply:
I like this idea. I'm going to track it and follow up with another PR. https://codedotorg.atlassian.net/browse/PLAT-1514